stl/inc/ranges

```diff
@@ -2115,51 +2114,6 @@ namespace ranges {
         };
 
         inline constexpr _Drop_fn drop;
-

```

Comment:
The PR will be a bit of a hassle, can we pull that out into a small cleanup PR?

Reply:
What, how where?